babel-watch.js

```diff
@@ -247,18 +249,21 @@ function restartApp() {
     }
   }
 
-  // Support for --debug option
   const runnerExecArgv = process.execArgv.slice();
-  if (program.debug) {
-    runnerExecArgv.push('--debug=' + program.debug);
-  }
-  // Support for --inspect option
-  if (program.inspect) {
-    runnerExecArgv.push('--inspect');
-  }
-  // Support for --debug-brk
-  if(program.debugBrk) {
-    runnerExecArgv.push('--debug-brk');
+  const unknownOptions = program.parseOptions(program.normalize(process.argv.slice(2))).unknown;
+
+  if (unknownOptions.length > 0) {
+    for (let i = 0, len = unknownOptions.length; i < len; ++i) {
```

Comment:
Can't we just replace this whole block and just pass all the unknown options directly to `runnerExecArgv`? If there are options like `--debug 2388` there is no need in changing that to `--debug=2388`, right?

If so this would greatly simplified this fragment as we could just do:
```
runnerExecArgv = runnerExecArgv.concat(unknownOptions);
```